cookbooks/aws-parallelcluster-scheduler-plugin/recipes/config_head_node.rb

```diff
@@ -38,3 +38,33 @@
 execute_event_handler 'HeadConfigure' do
   event_command(lazy { node['cluster']['config'].dig(:Scheduling, :SchedulerSettings, :SchedulerDefinition, :Events, :HeadConfigure, :ExecuteCommand, :Command) })
 end
+
+cookbook_file "#{node['cluster']['scripts_dir']}/get-compute-fleet-status.py" do
+  source 'compute_fleet_status/get-compute-fleet-status.py'
+  owner 'root'
+  group 'root'
+  mode '0755'
+  not_if { ::File.exist?("#{node['cluster']['scripts_dir']}/get-compute-fleet-status.py") }
+end
+
+cookbook_file "#{node['cluster']['scripts_dir']}/update-compute-fleet-status.py" do
+  source 'compute_fleet_status/update-compute-fleet-status.py'
+  owner 'root'
+  group 'root'
+  mode '0755'
+  not_if { ::File.exist?("#{node['cluster']['scripts_dir']}/update-compute-fleet-status.py") }
+end
+
+template "/usr/local/sbin/update-compute-fleet-status.sh" do
```

Comment:
change `/usr/local/sbin` to `/usr/local/bin` because this can be executed by non-root users e.g. scheduler-plugin user

Reply:
Changed